templates/course/length.php

```diff
@@ -19,4 +19,3 @@
 <div class="llms-meta llms-course-length">
 	<p><?php printf( __( 'Estimated Time: <span class="length">%s</span>', 'lifterlms' ), $course->get( 'length' ) ); ?></p>
 </div>
-
```

Comment:
You removed this on purpose?

Reply:
Yeah, I thought only one blank line is needed. So, I deleted the extra line.